test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyContractTestsSuite.java

```diff
@@ -1382,11 +1382,7 @@ private HapiSpec lazyCreateThroughPrecompileNotSupportedWhenFlagDisabled() {
                                                     .via(NOT_SUPPORTED_TXN)
                                                     .gas(GAS_TO_OFFER),
                                             getAliasedAccountInfo(SECP_256K1_SOURCE_KEY)
-                                                    .hasCostAnswerPrecheck(INVALID_ACCOUNT_ID),
-                                            childRecordsCheck(
-                                                    NOT_SUPPORTED_TXN,
-                                                    CONTRACT_REVERT_EXECUTED,
-                                                    recordWith().status(NOT_SUPPORTED)));
+                                                    .hasCostAnswerPrecheck(INVALID_ACCOUNT_ID));
```

Comment:
We should revisit this test case.  It doesn't depend on delegate call, the bytecode just happens to have outdated HederaTokenService.sol compiled in.